source/manual/alerts/email-alert-api-high-queue-latency.html.md

```diff
@@ -14,10 +14,44 @@ review_in: 6 months
 This means the time it takes for a job to be processed is unusually high. This
 may indicate a problem down the line which is preventing workers from being
 processed. It may also imply the threshold is too low if a large number of
-emails have being sent out due to a content change.
+emails are currently being processed.
+
+#### `delivery_transactional`
+
+The `delivery_transactional` queue is used for when we need to send one time
+emails. Some examples of one time emails could be a confirmation of a
+subscription or when validating a new subscribers email address.
+
+The [Sidekiq priority][priority] for this queue is weighted highly and there
+should only be a very small number of jobs on the queue at any one time which
+means that these emails should be processed pretty promptly. If this is not the
+case and this alert is firing then we should investigate.
+
+We should initially look for degradation of the service as high latency
+normally indicates degraded performance or high load, or both.
+
+Some diagnostic steps you could take are below:
+
+* Check to see if we have a high number of emails queued up. If these is a
+  large amount of `delivery_immediate_high` emails backed up (same Sidekiq
+  priority) then these transactional emails could be stuck waiting to be
+  processed.
+* If there is a high amount of emails in the queue, are we processing them at
+  a normal speed? Under high load we regularly send requests to Notify at a
+  rate of ~300 email requests per second.
+* Are the workers reporting any problems or any issues being raised in
+  [sentry][sentry]?
+* If everything looks okay from the above you could also check the Sidekiq
+  logs which are located on each email-alert-api machine here ->
+  `/var/apps/email-alert-api/sidekiq.json.log`
+* You could also look at the jobs in the queue directly to see if anything
```

Comment:
Not sure we'd gain anything here. We should have been able to diagnose the problem from the previous steps. The fix may involve doing something in the console, but it's hard to say what in advance. At least we can help people get to that point.

Reply:
Sure, I've removed this step.